.github/workflows/wpt-mapper.yml

```diff
@@ -45,29 +45,33 @@ jobs:
       - name: Run tests
         # For verbose logging, add --log-mach - --log-mach-level info
         run: >
-          ./wpt run
-          --webdriver-binary ../runBiDiServer.sh
+          ./wpt/wpt run
+          --webdriver-binary ./runBiDiServer.sh
           --binary /usr/bin/google-chrome-unstable
           --manifest MANIFEST.json
-          --metadata ../wpt-metadata/mapper
+          --metadata ./wpt-metadata/mapper
           --log-wptreport wptreport.json
           chromium
           webdriver/tests/bidi/
-        working-directory: wpt
       - name: Update expectations
         if: ${{ always() }}
         run: >
           ./wpt update-expectations
           --product chromium
           --manifest MANIFEST.json
-          --metadata ../wpt-metadata/mapper wptreport.json
-        working-directory: wpt
+          --metadata ./wpt-metadata/mapper wptreport.json
       - name: Upload expectations
         if: ${{ always() }}
         uses: actions/upload-artifact@v3
         with:
           name: wpt-metadata-mapper
           path: wpt-metadata/mapper
+      - name: Upload WPT run report
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: wptreport-mapper
+          path: wptreport.json
```

Comment:
FYI it's also possible to add multiple paths at once, e.g.

  - name: Upload expectations and report
        if: ${{ always() }}
        uses: actions/upload-artifact@v3
        with:
          name: wpt-mapper
          path: wpt-metadata/mapper
          path: wptreport.json

https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions

...although, LGTM as is either way.

Reply:
thanks for the update! Those artefacts are going to be used in different scenarios, so let's keep the split at least for now.